vissl/models/base_ssl_model.py

```diff
@@ -551,11 +551,11 @@ def init_model_from_weights_params_file(
     def is_clustering_model(self):
         """
         Whether or not the model is a clustering model
-        (only supports SwAV for the moment)
+        (supports SwAV and DeepCluster_v2 for the moment)
         """
-        return len(self.heads) == 1 and any(
-            isinstance(module, SwAVPrototypesHead) for module in self.heads[0].modules()
-        )
+        return len(self.heads) == 1 or any(
```

Comment:
This change doesn't really make sense to me -- can you help explain?

1. This doesn't look like it would be backwards compatible when using the SwAVPrototypesHead.
2. Here you are setting the cluster_assignment for different images to the same value?

For deepclusterv2, the clustering ocurrs in the loss. See `deepclusterv2_loss.py`.